src/glyph.rs

```diff
@@ -31,6 +32,34 @@ pub enum Segment {
     Cubic(Offset, Offset, Offset),
 }
 
+impl Segment {
+    /// Helper function to interpolate the current Segment
+    pub fn interpolate(&self, t: f32) -> Offset {
```

Comment:
I would rather call it `evaluate`. It is not like we have a choice here. It is the definition of the segment. What do you think?